flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/JobGraph.java

```diff
@@ -102,60 +101,150 @@
 	// --------------------------------------------------------------------------------------------
 
 	/**
-	 * Constructs a new job graph with no name and a random job ID.
+	 * Constructs a new job graph with no name, a random job ID, and the default
+	 * {@link ExecutionConfig} parameters.
 	 */
 	public JobGraph() {
 		this((String) null);
 	}
 
 	/**
-	 * Constructs a new job graph with the given name, a random job ID.
+	 * Constructs a new job graph with the given name, a random job ID and the default
+	 * {@link ExecutionConfig} parameters.
 	 *
 	 * @param jobName The name of the job
 	 */
 	public JobGraph(String jobName) {
-		this(null, jobName);
+		this(null, jobName, new ExecutionConfig());
```

Comment:
Again here the constructors that do not specify executionConfigs are only exists for tests. So here the empty ExecutionConfig is just to not break the already existing tests that were testing other parts of the functionality.